Cesium.CodeGen/Contexts/AssemblyContext.cs

```diff
@@ -73,8 +77,35 @@ private AssemblyContext(AssemblyDefinition assembly, ModuleDefinition module, As
                 module.Types.Add(type);
                 return type;
             });
-    }
+        Namespace = @namespace;
+
+        if (!string.IsNullOrWhiteSpace(globalTypeFQN))
+        {
+            string typeName;
+            string typeNamespace;
 
+            if (!globalTypeFQN.Contains('.'))
+            {
+                typeName = globalTypeFQN;
+                typeNamespace = @namespace;
+                GlobalTypeFQN = $"{@namespace}.{globalTypeFQN}";
+            }
```

Comment:
This I don't like. It makes impossible to define a namespace for all other types, and then mark the global type as namespace-less. In particular, I believe that `--namespace SomeNS --globalclass GlobalClass` should lead to the generation of a class with FQN `GlobalClass` (i.e. without a namespace).

This would be more flexible for various user demands.